.changeset/hungry-tables-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/react": patch
```

Comment:
I think you're good skipping a changeset for this type of change since we're not changing any part of the package 👍

Reply:
Ahh, I wasn't sure, I think last time I was asked to add one. But happy to remove.